symplyphysics/laws/kinematic/rotational_inertia_cartesian_integral.py

```diff
@@ -0,0 +1,49 @@
+from sympy import Integral
+from symplyphysics import (
+    units,
+    Symbol,
+    Quantity,
+    Function,
+    validate_input,
+    validate_output,
+)
+
+# Description
+## In case of a rigid body with a continuously distributed mass, its rotational inertia is expressed
+## as an integral over the entire body.
+
+# Law: I = Integral(rho(x, y, z) * (x**2 + y**2), dx*dy*dz)
```

Comment:
Probably it can be described more high level, like this:
`Integral(rho * r**2, dV)`

Reply:
Wouldn't this evaluate to `rho * r**2 * V` after integrating over `V`?

Reply:
Or you mean make both `rho` and `r` functions of `V`, and the end user can later do a substitution like `dV -> dx*dy*dz` and then compute the integral?

Reply:
Yes.
Well it's law who should calculate volume element. So user should only pass integration limits.